platform/jobs/edxTravisCounts.groovy

```diff
@@ -0,0 +1,57 @@
+package platform
+
+Map config = [:]
+Binding bindings = getBinding()
+config.putAll(bindings.getVariables())
+PrintStream out = config['out']
+
+// Travis Jobs
+
+// The travis-counts job
+Map prJob = [ name: 'travis-counts',
+              deprecated: false,
+              worker: 'coverage-worker',
+              branch: '*/master'
+              ]
+
+/* Iterate over the job configurations */
```

Comment:
I don't think we need to have any secrets (and therefore no assertions). This job doesn't contain any sensitive information.  You can write all the groovy code without any loop over secretMap/configs